Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
> You could just as easily cast to `id` and use the method, not the property, to silence any compiler warnings.

true…Maybe it's nice that you can opt-in and get some level of formality (so it's not an old-style [informal protocol](https://developer.apple.com/library/ios/documentation/General/Conceptual/DevPedia-CocoaCore/Protocol.html))? Then, if you do opt in, the compiler can check that you're actually implementing what's necessary _and_ can check this callsite.

meh, it is weird, though.